readme.md

```diff
@@ -3,4 +3,14 @@ __UNOFFICIAL__ API
 ## How To Use
  - Watch My Youtube Video https://www.youtube.com/watch?v=wGM9V9tHJ2o (OUTDATED)
  - You don't need all of things that are shown in the video. 
- - watch the video & get header cookie and new token(you can get the token where i took ASEC from. it's next to ASEC if you try now)
\ No newline at end of file
+ - watch the video & get header cookie and new token(you can get the token where i took ASEC from. it's next to ASEC if you try now)
+
+
+
+## Installations
+
+```bash
+python3 -m pip install git+git://github.com/Duerocraft/AternosAPI
+```
+
+(The library is not available on **pypy**).
```

Comment:
The package is already available on pypi

Reply:
Fixed. Thanks!